backend/rust/bunker/src/service.rs

```diff
@@ -0,0 +1,43 @@
+//! Contains the service trait for the bunker service.
+
+use crate::pb::Result as PbResult;
+use crate::pb::{
+    EmbeddingResult, GenerateImageRequest, HealthMessage, ModelOptions, PredictOptions, Reply,
+    StatusResponse, TokenizationResponse, TranscriptRequest, TranscriptResult, TtsRequest,
+};
+use async_trait::async_trait;
+use tokio_stream::wrappers::ReceiverStream;
+use tonic::{Request, Response, Status};
+
+#[async_trait]
+pub trait BackendService<T = ReceiverStream<Result<Reply, Status>>> {
```

Comment:
This is the same as the generated Backend, `pub use pb::backend_server::Backend as BackendService` maybe?

Reply:
They are the same thing. I have removed the `service.rs`. It does the same thing that the backend.rs already did. So, it is only redefining all the traits again. It is because I do missing understand "thin lib with  Backend trait."